lib/src/realm_class.dart

```diff
@@ -62,6 +63,23 @@ class Realm {
     }
   }
 
+  static bool existsSync(Configuration config) {
```

Comment:
I know we discussed passing a configuration here but on a second thought we should probably require a path here imo. It's somewhat unnatural to require a config which requires a schema to be constructed. So if we require a path and there is a config around one can do 
```Dart
Realm.existsSync(config.path);
```
or if there is not a config around they can do 
```Dart
Realm.existsSync(Configuration.defaultPath); //or pass a custom string here. 
```

I think its more natural and especially since this is a static method we shouldn't require a config which is somewhat an instance concept.

Reply:
@nielsenko  @desistefanova what do you think about this ^

Reply:
I have also changed my mind, and agree :-)

Reply:
It is a strong argument. I agree @blagoev.

Reply:
Fixed in https://github.com/realm/realm-dart/pull/171/commits/36ef83d9efb04bc3700725948649336692d23e86